Dockerfile

```diff
@@ -16,19 +16,19 @@
 # WARNING: THIS DOCKERFILE IS NOT INTENDED FOR PRODUCTION USE OR DEPLOYMENT.
```

Comment:
:)

Reply:
Not yet. I left it in until we get confidence :)